src/index.test.js

```diff
@@ -23,33 +23,44 @@ describe('init', () => {
       value: spy,
       writeable: true,
     });
-
-    // Result
-    result = init(KEY);
```

Comment:
Now that there is a defaultConfig, you can put this line back and remove it from all the places where you added it except for the last where you added a custom config object

Reply:
result = init(key) was taken out from beforeEach in order not to init() method with default configuration in "should not attach a click event listener to the window" test. Else it will be invoked and once before configuration is set to false and expect(spy).not.toHaveBeenCalled(); will fail

Reply:
👍